updated the readme.md (#36)
flatcar/flatcar-terraform: README.md

[![Mastodon Follow](https://img.shields.io/badge/Mastodon-Follow-6364FF?logo=mastodon)](https://hachyderm.io/@flatcar)
[![Bluesky](https://img.shields.io/badge/Bluesky-Follow-0285FF?logo=bluesky)](https://bsky.app/profile/flatcar.org)

</div> 

# Table of Contents

- [flatcar-terraform](#flatcar-terraform)
  - [AWS EKS worker nodes](#aws-eks-worker-nodes)
  - [Plain Flatcar instances](#plain-flatcar-instances)

>     - [digitalocean](#digitalocean)

Reviewer: I _personally_ find this approach a bit less intuitive: one need to click here, to be redirected to L46-47 and one must click again on L46 to actually reach the `digitalocean` folder.

What about simply provide a list of all available examples in alphabetic order:
```md
# Available examples
- [aws](aws)
- ...

    - [equinix-metal-aka-packet](#equinix-metal-aka-packet)
    - [flatcar-terraform-hetzner](#flatcar-terraform-hetzner)
    - [qemu-libvirt](#qemu-libvirt)
    - [openstack](#openstack)
  - [Experiments for re-running Ignition instead of instance replacement on userdata changes](#experiments-for-re-running-ignition-instead-of-instance-replacement-on-userdata-changes)
    - [equinix-metal-aka-packet-without-instance-replacement](#equinix-metal-aka-packet-without-instance-replacement)
    - [qemu-libvirt-without-instance-replacement](#qemu-libvirt-without-instance-replacement)

> ## flatcar-terraform

Reviewer: I would keep this as it is right now to have the following structure:
```md
## flatcar-terraform

Examples of deploying Flatcar instances with Terraform/OpenTofu. Follow the README instructions in the directories to try it out.
<!-- The warning -->

# Available examples
- [aws](aws)
...
```

Examples of deploying Flatcar instances with Terraform

> :warning: This is really for demo purposes but it can serve as a foundation (for example do not pass the admin configuration through HTTP for workers to join) :warning:

Reviewer: ```suggestion
:warning: This is really for demo purposes: do not use those examples in production :warning:
```
